package.json

```diff
@@ -2,17 +2,19 @@
   "name": "slimevr-ui",
   "version": "0.5.1",
   "private": true,
+  "packageManager": "pnpm@8.7.6",
```

Comment:
```suggestion
  "packageManager": "pnpm@8.10.2",
```

Reply:
This happened to me in my windows machine https://github.com/pnpm/pnpm/issues/7213